Classes/UVSuggestionListViewController.m

```diff
@@ -346,19 +269,29 @@ - (void)initNavigationItem {
     }
 }
 
+- (void)dismiss {
```

Comment:
When I initially started making the changes for the UISearchController I had to add that method b/c it would not dismiss unless I set the searchResultsController to nil when it was called. This was probably due to having something coded wrong initially. I removed that method call now after making all of the recent changes and it seems to behave as expected without it. I pushed the change up just now.